.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,35 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+
+      - name: Setup npm registry
+        env:
+          NPM_TOKEN: ${{ secrets.DEPLOY_NPM_TOKEN }}
+        run: |
+          echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" \
+            > $HOME/.npmrc
+
+      - name: Kyber
+        run: |
+          cd external/js/kyber &&
```

Comment:
I do not think that `&&`s are necessary in multi-line commands, as shells seem to be run with `-e` according to [this](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell).
Might be worth to specify `shell: bash` to be explicit, though.

Reply:
true - that must be some copy/paste from a Makefile or so...